arbitrum-docs/for-devs/quickstart-solidity-hardhat.md

```diff
@@ -43,7 +43,7 @@ Let's implement the "web3" version of the above vending machine using Arbitrum.
 ### Prerequisites
 
 - **VS Code**: The IDE we'll use to build our vending machine. See [code.visualstudio.com](https://code.visualstudio.com/) to install.
-- **Metamask**: The <a data-quicklook-from='ethereum-wallet'>wallet</a> we'll use to interact with our vending machine. See [metamask.io](https://metamask.io/) to install.
+- **Metamask**: The <a data-quicklook-from='ethereum-wallet'>wallet</a> we'll use to interact with our vending machine. See [metamask.io](https://metamask.io/) to install. To be able to reach the end of this tutorial, you will need minimum funds ($ETH) available in your wallet.
```

Comment:
```suggestion
- **Metamask**: The <a data-quicklook-from='ethereum-wallet'>wallet</a> we'll use to interact with our vending machine. See [metamask.io](https://metamask.io/) to install.
```

Thanks for this! Recommend keeping this concise to minimize onboarding friction/intimidation - will update the "We'll install the rest of our dependencies as we go" to set expectations more clearly.